### ISSUE TITLE
cloudformation deploy regression in latest version

### ISSUE THREAD

**Issue by zkhan96MSM** [localstack/localstack#2336]
# Type of request: This is a ...

[x] bug report
[ ] feature request

# Detailed description
I have been doing an `aws cloudformation deploy` command on the latest version of localstack. However I get an error when this happens, when rolling back to version 0.9.4, the deploy works.

## Expected behavior
Using v0.9.4
```
aws cloudformation deploy \
      --template-file cloudformation.json \
      --stack-name some-stack \
      --endpoint "http://localhost:4581"

Waiting for changeset to be created..
Waiting for stack create/update to complete
Successfully created/updated stack - some-stack
```

## Actual behavior / Steps to reproduce
```
aws cloudformation deploy \
      --template-file cloudformation.json \
      --stack-name some-stack \
      --endpoint "http://localhost:4566"        

Waiting for changeset to be created..

'Status'

```

This creates the stack, and changeset, all looks good.

Except for the fact that the cloudformation.json file is creating a dynamoDB table, and no tables are created:
```
aws dynamodb list-tables --endpoint http://localhost:4566 | cat
{
    "TableNames": []
}
```

Running the same cloudformation command again returns the following error:
```
 aws cloudformation deploy \                                     
      --template-file cloudformation.json \
      --stack-name some-stack \
      --endpoint "http://localhost:4566"

Unable to parse response (syntax error: line 1, column 54), invalid XML received:
b'<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n<title>500 Internal Server Error</title>\n<h1>Internal Server Error</h1>\n<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>\n'
```

## Command used to start LocalStack

`TMPDIR=/private$TMPDIR docker-compose up -d`

Using the `docker-compose.yml` in the latest version of the repo 

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
```
aws cloudformation deploy \                                     
      --template-file cloudformation.json \
      --stack-name some-stack \
      --endpoint "http://localhost:4566"
```

This is the `cloudformation.json`:

```
{
  "Resources": {
    "SomeTable": {
      "Type": "AWS::DynamoDB::Table",
      "Properties": {
        "AttributeDefinitions": [
          {
            "AttributeName": "name",
            "AttributeType": "S"
          },
          {
            "AttributeName": "id",
            "AttributeType": "N"
          }
        ],
        "KeySchema": [
          {
            "AttributeName": "id",
            "KeyType": "HASH"
          }
        ],
        "GlobalSecondaryIndexes": [
          {
            "IndexName": "nameIndex",
            "KeySchema": [
              {
                "AttributeName": "name",
                "KeyType": "HASH"
              }
            ],
            "Projection": {
              "ProjectionType": "ALL"
            },
            "ProvisionedThroughput": {
              "ReadCapacityUnits": 5,
              "WriteCapacityUnits": 5
            }
          }
        ],
        "BillingMode": "PROVISIONED",
        "ProvisionedThroughput": {
          "ReadCapacityUnits": 5,
          "WriteCapacityUnits": 5
        },
        "TableName": "some_table"
      }
    }
  }
}
```

**Comment by rakcy** [comment]
This looks like it's related to this issue:
https://github.com/localstack/localstack/issues/2528

**Comment by whummer** [comment]
Thanks for reporting @zkhan96MSM @rakcy . This issue should be fixed in #2644 - can you please give it another try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks
